test/integration/wallet/wallet.test.ts

```diff
@@ -1,10 +1,10 @@
 import { UnimplementedMethodError } from '@liquality/errors';
```

Comment:
after changing the whole wallet test suite, can you run the tests for the other networks too, please?

Reply:
Yeah working fine except for BTC:

<img width="544" alt="image" src="https://user-images.githubusercontent.com/32274987/163840539-05ccbd7f-5213-434e-b959-9d795a3d4c32.png">

 Is there any additional step that i need to do in order to execute these as well?

Reply:
oh, you need docker for BTC... instead of running `yarn test:integration` you can run `yarn test` which will setup the local BTC node. Make sure you have docker installed first.